types/markdown-it/tslint.json

```diff
@@ -3,9 +3,7 @@
     "rules": {
         "array-type": false,
         "comment-format": false,
-        "dt-header": false,
```

Comment:
why this rule removed?

Reply:
I've corrected header to be aligned with current DT tooling (`v12.2.0` > `12.2`)
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library
I can retract this change if you with, but we promote adhering to 'default' configurations across packages as much as possible, to ease contribution and maintenance, thx!